### PR TITLE
[documentation] bugfix: zsh directory not found

### PR DESCRIPTION
**Pull request by lusantisuper**
In this PR, I fixed the issue that zsh can't find directory to build hal.

### CODE REVIEW

File: 6-contrib/1-build.md [hunk 1]
```diff
@@ -17,8 +17,8 @@ git clone --recursive https://github.com/nanvix/hal.git # Clone the source tree
 Install build dependencies.
 
 ```
-cd $WORKDIR/hal/utils                   # Enter the source tree
-sudo bash nanvix-setup-prerequisites.sh # Get essential tools for building
+cd $WORKDIR'/hal/utils'                   # Enter the source tree
+sudo bash nanvix-setup-prerequisites.sh   # Get essential tools for building
```
Comment (ppenna) [inline 6-contrib/1-build.md:21]: Watch out for trailing white spaces. We don't need to touch this line ;-)

File: 6-contrib/1-build.md [hunk 2]
```diff
@@ -17,8 +17,8 @@ git clone --recursive https://github.com/nanvix/hal.git # Clone the source tree
 Install build dependencies.
 
 ```
-cd $WORKDIR/hal/utils                   # Enter the source tree
-sudo bash nanvix-setup-prerequisites.sh # Get essential tools for building
+cd $WORKDIR'/hal/utils'                   # Enter the source tree
```
Comment (ppenna) [inline 6-contrib/1-build.md:20]: Could we escape the whole argument? It reads cleaner

File: 6-contrib/1-build.md [hunk 3]
```diff
@@ -49,13 +49,13 @@ bash nanvix-setup-qemu.sh
 Add simulators to your path:
 
 ```
-export PATH=$PATH:$WORKDIR/hal/utils/toolchain/qemu/bin
+export PATH=$PATH':'$WORKDIR'/hal/utils/toolchain/qemu/bin'
```
Comment (ppenna) [inline 6-contrib/1-build.md:52]: Could we escape the whole argument? It reads cleaner

File: 6-contrib/1-build.md [hunk 4]
```diff
@@ -49,13 +49,13 @@ bash nanvix-setup-qemu.sh
...
 ```
 
 **3. Build**
 
 ```
-cd $WORDKIR/hal
+cd $WORKDIR'/hal'
```
Comment (ppenna) [inline 6-contrib/1-build.md:58]: Could we escape the whole argument? It reads cleaner

**Review by ppenna — CHANGES_REQUESTED** [review]
Some small fixes to go

File: 6-contrib/1-build.md (hunk 2 above)
Comment (lusantisuper) [reply 6-contrib/1-build.md:20]: ZSH needs explicity separation from ambient variable to strings.

File: 6-contrib/1-build.md (hunk 1 above)
Comment (lusantisuper) [reply 6-contrib/1-build.md:21]: This space is only to match the identation of the other line.

File: 6-contrib/1-build.md (hunk 3 above)
Comment (lusantisuper) [reply 6-contrib/1-build.md:52]: ZSH needs explicity separation from ambient variable to strings.

File: 6-contrib/1-build.md (hunk 4 above)
Comment (lusantisuper) [reply 6-contrib/1-build.md:58]: ZSH needs explicity separation from ambient variable to strings.

**Review by ppenna — APPROVED** [review]

**Review by ppenna — APPROVED** [review]

**Review by ppenna — APPROVED** [review]

**Pull request merged**